deliver/lib/deliver/options.rb

```diff
@@ -159,6 +159,11 @@ def self.available_options
                                      env_name: "DELIVER_ITC_PROVIDER",
                                      description: "The provider short name to be used with the iTMSTransporter to identify your team",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :overwrite_screenshots,
+                                     env_name: "DELIVER_OVERWRITE_SCREENSHOTS",
+                                     description: "Clear all previously uploaded screenshots",
```

Comment:
Let's use `Clear all previously uploaded screenshots before uploading the new ones`